apps/root/src/pages/position-detail/components/summary-container/index.tsx

```diff
@@ -53,9 +155,25 @@ const PositionSummaryContainer = ({ position, pendingTransaction, isLoading }: P
             <GraphContainer position={position} isLoading={isLoading} />
           </Grid>
           <Grid item xs={12}>
-            <StyledPaper>
-              <PositionSwaps position={position} isLoading={isLoading} />
-            </StyledPaper>
+            <PositionTimelineControls
+              items={timelineItems}
+              renderContent={(item) =>
+                MESSAGE_MAP[item.positionState.action](item.positionState, item.position).content
+              }
+              renderHeader={(item) =>
+                buildDcaTimelineHeader(
+                  MESSAGE_MAP[item.positionState.action](item.positionState, item.position).title,
+                  item.positionState,
+                  item.position.chainId
+                )
+              }
+              renderIcon={(item) => MESSAGE_MAP[item.positionState.action](item.positionState, item.position).icon}
+              getItemId={(item) => `${item.position.chainId}-${item.positionState.tx.hash}`}
```

Comment:
this could be defined inside the item?

Reply:
Yes, actually it was implemented as you mention. Since the same method applies to all items, I thought it would be easier to implement it here rather than adding it to each item individually